mps_youtube/util.py

```diff
@@ -518,7 +518,7 @@ def _get_metadata_from_lastfm(artist, track):
 
     metadata = dict()
 
-    data = json.loads(resp.read())
+    data = json.loads(resp.read().decode('utf-8'))
```

Comment:
It seems like the `HTTPResponse` returned by `urlopen` can be passed to `json.load`. That would probably be better...

Reply:
I have made the change.